pkg/frontend/admin_openshiftcluster_etcdcertificaterenew.go

```diff
@@ -299,6 +303,47 @@ func (e *etcdrenew) validateEtcdCertsExistsAndExpiry(ctx context.Context) error
 	return nil
 }
 
+func (e *etcdrenew) validateEtcdCertsRenewed(ctx context.Context) error {
+	e.log.Infoln("validating if etcd certs are renewed")
+	isError := false
+
+	for _, secretname := range e.secretNames {
+		e.log.Infof("validating secret %s", secretname)
+		cert, err := e.k.KubeGet(ctx, "Secret", namespaceEtcds, secretname)
+		if err != nil {
+			return err
+		}
+
+		var u unstructured.Unstructured
+		var secret corev1.Secret
+		if err = json.Unmarshal(cert, &u); err != nil {
+			return err
+		}
+		err = kruntime.DefaultUnstructuredConverter.FromUnstructured(u.Object, &secret)
+		if err != nil {
+			return err
+		}
+		_, certData, err := utilpem.Parse(secret.Data[corev1.TLSCertKey])
+		if err != nil {
+			return err
+		}
+
+		// etcd operator renews certificates for another 3 years, 1000+ days (3*365)
+		e.log.Infof("certificate '%s' expiration date is '%s'", secretname, certData[0].NotAfter)
```

Comment:
`certData[0]` is a potential index out of range error. We're not testing to make sure there is at least 1 item in the certData array.